### PR TITLE
Config bug fix

### PR DESCRIPTION
**Pull request by spendres**
Simple bug that converted errors like "duplicate MACH setting in configuration file" to "missing configuration file".

### CODE REVIEW

File: SU2_PY/SU2/io/config.py [hunk 1]
```diff
@@ -85,9 +85,12 @@ def __init__(self,*args,**kwarg):
         # read config if it exists
         if filename:
             try:
-                self.read(filename)
-            except:
-                raise IOError , 'Could not find config file: %s' % filename
+                self.read(filename)	# is really read with IO Exceptions, and process with Asserts
+            except IOError:
+                print 'Could not find config file: %s' % filename
+	    except:			# for asserts that test validity of user modified config file
```
Comment (aerialhedgehog) [inline SU2_PY/SU2/io/config.py]: this block isn't aligning with the rest of the statement. make sure you're using spaces too!

**Comment by aerialhedgehog** [comment]
thanks for the commit ! take a look a the comment, one quick fix needed
- trent

**Comment by aerialhedgehog** [comment]
so now the file is being committed as deleted...  please roll back to this commit
https://github.com/spendres/SU2/commit/6cbddbe59cf5da89e89321b72ae20ab289a66520

here are the commands you can use --

# Reset the index to the desired tree
git reset 6cbddbe59cf5da89e89321b72ae20ab289a66520

# Move the branch pointer back to the previous HEAD
git reset --soft HEAD@{1}

git commit -m "Revert to 6cbddbe"

# Update working copy to reflect the new commit
git reset --hard

**Comment by spendres** [comment]
thanks for the commands

**Comment by spendres** [comment]
By the way, I have another set up updates for you guys to consider that are
beyond the scope of this pull request. Let me know if you are waiting on me
to close the pull request or if you can do that.

I put the recommended changes for the structure of config.py file in a gist
<https://gist.github.com/spendres/7bbe6eb60fac1f2f148a> that separates the
low level config errors validation. I also created a few unit test cases to
allow a controlled build up of regex checks without risk of breaking
previous bug fixes.

Let me know

Steve

**Comment by fpalacios** [comment]
Hi Steve,

I think that all your contributions are very interesting, so please feel free to create a pull request.

Only one comment, please be sure that you pull to the develop branch (not the master). We only use the master branch for releasing the code.

Thanks again!

Best
Francisco

**Comment by hlkline** [comment]
I have closed this pull request because it needs to be switched to the develop branch instead of the master branch. 
Steve, if interested please go ahead and create a new pull request with your changes going into the develop branch. 

Thanks!

**Pull request closed without merging**